### PR TITLE
Implement collapse/expand for course info

### PR DESCRIPTION
**Pull request by noisecapella**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
  - [x] Tag @ferdi or @pdpinch for review
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Part of #312 

#### What's this PR do?
Adds a button to toggle course info

#### How should this be manually tested?
Click the expander button

#### Screenshots (if appropriate)
Desktop:
Screenshot from 2020-11-20 18-15-09
Screenshot from 2020-11-20 18-15-21

Mobile:
Screenshot_2020-11-20 Course Home Computer System Engineering Electrical Engineering and Computer Science MIT OpenCourseWare
Screenshot_2020-11-20 Course Home Computer System Engineering Electrical Engineering and Computer Science MIT OpenCourseWare(1)

### CODE REVIEW

**Comment by aliceriot** [comment]
two things I noticed / questions:

one the home page the click target for expanding / contracting the course info panel is just the little chevron:

Screenshot from 2020-11-23 10-17-46

would it make sense for it to be expanded a little bit, to be the title as well?

**Comment by aliceriot** [comment]
on course section pages `COURSE INFO` is being forced on to two lines at desktop width

**Comment by noisecapella** [comment]
@abdulkdawson any thoughts on having the "Course Info" title be part of the link to expand/collapse? https://github.com/mitodl/hugo-course-publisher/pull/396#issuecomment-732227484

**Comment by abdulkdawson** [comment]
@noisecapella It wouldn't hurt to have the "Course Info" title as part of the link to expand/collapse.

**Comment by noisecapella** [comment]
@aliceriot I removed the expand/collapse from the sidebar altogether since it wasn't part of the issue

File: src/js/course_expander.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        if (event.key === "Enter") {
+          event.preventDefault()
+          toggleExpand(expanderButton, container)
+        }
+      })
+    }
+  }
+}
+
+export { initCourseInfoExpander }
```
Comment (aliceriot) [inline src/js/course_expander.js:32]: nice, jquery free!

**Review by aliceriot — APPROVED** [review]
LGTM! assuming the build is passing

**Pull request merged**